src/Observable.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { Observable } from './Observable'
+
+const impossibru = function (done, msg): () => void {
+    return (): void => {
+        expect(`${msg} WILL NOT HAPPEN`).toBe(''); setTimeout(done)
+    }
+}
+
+describe('Observable', () => {
+    it('can filter messages', done => {
+        const tester = value => {
+            expect(value).toBe(3)
+            setTimeout(done)
```

Comment:
[hint]

Tak tylko, żeby wyjaśnić.
setTimeout(done) jest po to, bo done od razu, w miejscu wywołania uruchamia następny test.
Normalnie to nie problem - Jasmine przełącza kontekst na inny test i wszystko śmiga.
Tutaj jest problem, bo będąc w callbacku na subscribe jesteś w fazie trawienia, w funkcji objętej Dispatcherem, więc wszystko co używa dispatchera będzie zdeferowane. To nie jest tragedia, ale łatwiej pisze się testy, jeżeli test nie jest w kolejce Dispatchera. Dodatkowo, jakby coś poszło nie tak, to zobaczyłbyś błąd w następnym teście, a nie tym, co też jest cięższe do ogarnięcia.
Tłumaczę, bo to już bebechy i Jasmine i Immview, więc niełatwo byłoby pewnie do tego dojść.